commands/flags.go

```
@@ -57,18 +61,33 @@ func parseOptions(cliContext *cli.Context) (*options.Options, error) {
 	}
 	logger.SetLevel(level)
 
-	ports := cliContext.IntSlice("port")
-	if len(ports) == 0 {
-		return nil, MissingParam(portFlag.Name)
-	}
-
 	listener := cliContext.String("listener")
 	if listener == "" {
 		return nil, MissingParam(listenerFlag.Name)
 	}
 
+	checksFile := cliContext.String("checks")
+	if checksFile == "" {
+		return nil, MissingParam(checksFlag.Name)
+	}
+	checksFileContents, err := ioutil.ReadFile(checksFile)
+	if err != nil {
+		fmt.Print(err)
```

In Go, you should return an error whenever you hit one, not just log it.